sisl/viz/plotly/plots/bands.py

```diff
@@ -865,3 +888,67 @@ def _plot_Kdiff(self, band1, band2, E=None, offsetE=False):
         plt.update_layout({"title": f"Delta K between bands {band1} and {band2}", 'xaxis_title': 'Delta k', 'yaxis_title': 'Energy [eV]'})
 
         return plt
+
+    def effective_mass(self, band, k, band_spin=0, n_points=10, k_direction="symmetric"):
+        """
```

Comment:
Remove newline

Reply:
What do you mean? No empty line between methods? :sweat_smile:

Reply:
In documentation. The first line is a special line for docs...

Reply:
Aah ok, sorry! :sweat_smile:

Reply:
I'll have to do a PR to correct this basically everywhere then :facepalm:

Reply:
Yeah, I have noticed ;) I corrected many in the initial merge. But stopped at some point. I will have to see if it actually matters, but I think it does for instance for `help` calls etc. Everywhere I find that the first line is _special_.